docsets/pymc/docset.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "PyMC",
-  "version": "5.9.0",
+  "version": "5.14.0",
   "archive": "PyMC.tgz",
```

Comment:
You need to update this file as well.

Reply:
You need to update this file as well. The main version archives is separate from the specific versions.